packages/blade/fonts.css

```diff
@@ -0,0 +1,170 @@
+@font-face {
+  font-family: 'Tasa Orbiter';
+  src: url('./fonts/tasa-orbiter-vf.woff2') format('woff2-variations');
+  font-weight: 125 950;
+  font-stretch: 75% 125%;
+  font-style: normal;
+  font-display: swap;
+}
+
+/* inter-cyrillic-ext-400-normal */
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  font-display: swap;
+  font-weight: 400;
+  src: url(./fonts/Inter/inter-cyrillic-ext-400-normal.woff2) format('woff2'),
```

Comment:
Its easier to ask consumers to just import `@razorpay/blade/fonts.css` and that takes care of all fonts rather than asking them to install 2 fonts from different files.